appengine/flexible/F#/F#.fsproj

```diff
@@ -11,9 +11,6 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Google.Cloud.Vision.V1" Version="1.0.0-beta05" />

```

Comment:
Can you update this to 2.0.0? Depending on what Vision is being used for, a couple of changes might be needed. Thanks.
https://www.nuget.org/packages/Google.Cloud.Vision.V1/

Reply:
Done.